auth/src/App.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import "./App.css";
+import { Route, NavLink } from "react-router-dom";
+import Login from "./Login";
+import Users from "./Users";
+import axios from "axios";
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      url: "http://localhost:5111"
+    };
+  }
+
+  login = user => {
+    console.log("hit login");
+    console.log("user", user);
+    // const axiosObj = { headers: user, body: {} };
+
+    axios.interceptors.request.use(
+      options => {
+        options.headers.username = user.username;
+        options.headers.password = user.password;
+        return options;
+      },
+      err => {
+        return Promise.reject(err);
+      }
+    );
+
+    // axios({
+    //   method: "post",
+    //   url: `${this.state.url}/api/login`,
+    //   headers: user
+    // })
+
+    axios
+      .get(`${this.state.url}/api/login`)
```

Comment:
good job implementing the use of the headers for the login. Do note that this system was used for instructional purposes to introduce using headers and most login systems you find will be using the req.body and a post method, reserving the headers for the passing of the jwt and thats about it